scipy/stats/_distn_infrastructure.py

```diff
@@ -2853,6 +2854,108 @@ def entropy(pk, qk=None, base=None, axis=0):
     return S
 
 
+def differential_entropy(
```

Comment:
This should probably go in `stats.py` or `morestats.py` rather than `_distn_infrastructure.py`.

Reply:
@tupui what do you think about this? 

I think we should actually move `entropy` and `differential_entropy` into `morestats.py`, but I can do it in an immediate followup to keep things cleaner.

Reply:
I agree with you, it seems like a better place for both methods! But yes, maybe better in another PR as this might require more work on the infrastructure which is not really related to this.